src/api/wayfire/idle.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include <wayfire/util.hpp>
+#include <wayfire/nonstd/wlroots-full.hpp>
+
+namespace wf
+{
+class idle_inhibitor_t
```

Comment:
Please add a public comment explaining what and how this is useful (you already explained it in the comment)